### ISSUE TITLE
[Android] Image file size of 0 with Camera

### ISSUE THREAD

**Issue by SY102134** [react-community/react-native-image-picker#532]
Android4.4.2
picker0.26.2

I found that every time launch camera, will create a 0 size image in Android/Data/xxx/Pictures/image-xxx.jpg
When we finished taking pictures ,will write data in this 0 size image, if cancelled the process {didCancel=true}, but this 0 size image is not deleted

**Comment by rusfearuth** [comment]
@wuhenwuya Does it happen when you cancel taken photo?

**Comment by wuhenwuya** [comment]
yes, empty image file not delete

**Comment by marcshilling** [comment]
Duplicate of #254

**Comment by wuhenwuya** [comment]
@marcshilling i find that throws an exception no delete image in catch

**Comment by rusfearuth** [comment]
@wuhenwuya It'll be fixed at next release. I've add fix for that bug in my local version of repository.

**Comment by wuhenwuya** [comment]
@rusfearuth Thinks, If you could, i want to write the image url into the system library, when we open the image library by image picker, we can see the pictures we took before; but now i can not see!

**Comment by rusfearuth** [comment]
@wuhenwuya Now, it's available from `develop` branch. I mean `writing the image url into the system library`

**Comment by rusfearuth** [comment]
@wuhenwuya Could you check it out? We've rolled out 0.26.3 release.

**Comment by wuhenwuya** [comment]
@rusfearuth very good，no problem in android4.4.2, image is also written to the system path; but have some minor problems:

1. because we first created 0 size image, if we finished taking pictures, but We do not click the Save button, but forced to close the app, or 0 size image still exist. so we can create image file after on press save button.
2. Image storage path in sdcard1/Pictures/image-xxx-xxx.jpg, not in app directory.
